osu.Game.Rulesets.Osu/Mods/OsuModObjectScaleTween.cs

```diff
@@ -23,7 +23,7 @@ public abstract class OsuModObjectScaleTween : Mod, IReadFromConfig, IApplicable
 
         public override double ScoreMultiplier => 1;
 
-        protected virtual float StartScale => 1;
+        public abstract BindableNumber<float> StartScale { get; }
```

Comment:
This could have stayed `protected` and just delegated to the bindable's `Value` in grow/shrink mods.

Reply:
I made the StartScale public & abstract as those were the modifiers used by `ModRateAdjust` that also uses a slider for changing its value, I could also not get it to work with `protected` as the customization button was grayed while using it, I'm not quite sure what to edit here

Reply:
Right now it doesn't matter too much. If [this suggestion](https://github.com/ppy/osu/pull/9595#discussion_r456827279) is deemed the correct direction then my proposal will make more sense. You would do

```csharp
public BindableNumber<float> StartScaleSetting { get; }
protected override float StartScale => StartScaleSetting.Value;
```

Reply:
I think we can go with this as a starting point and tweak the UX at a later stage.